src/test/kotlin/io/github/anki/anki/service/CardsServiceTest.kt

```diff
@@ -290,7 +316,10 @@ class CardsServiceTest {
             validateValidateUserHasPermissionsWasCalled()
 
             verify(exactly = 1) {
-                cardRepository.softDelete(initialMongoCard.id!!)
+                natsClient.request(
```

Comment:
it is not necessary because you mock it. I mean if it would not be mocked your test should fail.

Reply:
yes but i want to verify that this method was called and only once
if mock is configured but method was not called so i will not see any fails or even warnings